### PR TITLE
[cleanup] silence compiler warnings

### PR DESCRIPTION
**Pull request by Rechi**
## Description
Silence compiler warnings.

## Motivation and Context
I don't like those warnings.

## How Has This Been Tested?
compiled for android, osx and windows

## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: xbmc/windowing/android/AndroidUtils.cpp [hunk 1]
```diff
@@ -251,7 +251,7 @@ bool CAndroidUtils::SetNativeResolution(const RESOLUTION_INFO &res)
     CXBMCApp::SetDisplayMode(atoi(res.strId.c_str()));
     s_res_cur_displayMode = res;
   }
-  else if (abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
+  else if (std::abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
```
Comment (garbear) [inline xbmc/windowing/android/AndroidUtils.cpp]: std::fabs? also missing `#include <cmath>` I think

Comment (Rechi) [inline xbmc/windowing/android/AndroidUtils.cpp]: Doesn't `std::abs` have a float version (http://en.cppreference.com/w/cpp/numeric/math/fabs)?
Should I include `cmath` in this file also it currently compiles?

Comment (garbear) [inline xbmc/windowing/android/AndroidUtils.cpp]: OK, `std::abs` is fine. You should include cmath in case another include is removed that breaks the code.

**Comment by Rechi** [comment]
any objections?

**Review by FernetMenta — APPROVED** [review]

**Comment by AchimTuran** [comment]
I'm fine with your changes too. Thank you for reducing the amount of warnings.

**Review by AchimTuran — APPROVED** [review]

**Pull request merged**